release/cli/pkg/operations/testdata/main-bundle-release.yaml

```diff
@@ -86,17 +86,9 @@ spec:
         name: cert-manager-controller
         os: linux
         uri: public.ecr.aws/release-container-registry/cert-manager/cert-manager-controller:v1.14.7-eks-a-v0.0.0-dev-build.1
-      ctl:
-        arch:
-        - amd64
-        - arm64
-        description: Container image for cert-manager-ctl image
-        imageDigest: sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
-        name: cert-manager-ctl
-        os: linux
-        uri: public.ecr.aws/release-container-registry/cert-manager/cert-manager-ctl:v1.14.7-eks-a-v0.0.0-dev-build.1
       manifest:
         uri: https://release-bucket/artifacts/v0.0.0-dev-build.0/cert-manager/manifests/v1.14.7/cert-manager.yaml
+      startupapicheck: {}
```

Comment:
You'll need to add the `startupapicheck` image [here](https://github.com/aws/eks-anywhere/blob/main/release/cli/pkg/assets/config/bundle_release.go#L73) for these to not show up as empty. Basically if these testdata files have an empty struct, it means the actual bundle will have that empty struct too.